content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -15,6 +15,17 @@ This document serves both as a reference to the values and as a coordination poi
 
 ## Labels, annotations and taints used on API objects
 
+
+### apf.kubernetes.io/autoupdate-spec
+
+Type: Annotation
+
+Example: `apf.kubernetes.io/autoupdate-spec: true`
+
+Used on: [All APF configuration objects](/concepts/cluster-administration/flow-control/#defaults)
+
+Depending on whether this annotation is set to true or false, the object is controlled by the kube-apiservers or the user.
```

Comment:
This is not clear.
If this annotation is set to true, then the corresponding object is controlled by the API server?